test/unittests/t_dns.cc

```diff
@@ -953,6 +978,63 @@ TEST_F(T_Dns, HostfileResolverMultipleAddresses) {
   EXPECT_EQ(host.ipv6_addresses(), expected_ipv6);
 }
 
+TEST_F(T_Dns, HostfileResolverBlankLines) {
+  CreateHostfile("   \n  #comment\n127.0.0.1 localhost\n\n");
+  Host host = hostfile_resolver->Resolve("localhost");
+  EXPECT_EQ(kFailOk, host.status());
+  set<string> expected_ipv4;
+  expected_ipv4.insert("127.0.0.1");
+  EXPECT_EQ(host.ipv4_addresses(), expected_ipv4);
+}
+
+TEST_F(T_Dns, HostfileResolverTooLong) {
+  SetAltLogFunc(TDnsAltLogFunc);
```

Comment:
We just need to set it back to the default at the end of the test in order to not change the environment for following tests.  I'll take care of that.